resolwe/flow/executors/__init__.py

```diff
@@ -123,28 +125,41 @@ def update_data_status(self, **kwargs):
         try:
             # Ensure that we only update the fields that were changed.
             data.save(update_fields=update_fields)
-        except ValidationError as ex:
-            data.process_error.append(str(ex))
+
+            if kwargs.get('status', None) == Data.STATUS_ERROR:
+                self.failed = True
+                logger.error(
+                    __("Error occured while running a '{}' process.", self.process.name),
+                    extra={'data_id': self.data_id}
```

Comment:
I like that you report the data ID. Should we also include a link to the data object on the REST endpoint? It will save us some time if we'll be able to navigate to the data object with one click.